org.gnome.Epiphany.json

```diff
@@ -144,6 +144,22 @@
                 }
             ]
         },
+        {
+           "name": "gstreamer",
+             "cleanup": [
+               "*"
+          ],
+           "config-opts": [
+              "-Dvaapi=enabled"
```

Comment:
Here are the possible flags https://gitlab.freedesktop.org/gstreamer/gstreamer/-/blob/main/subprojects/gstreamer-vaapi/meson_options.txt?ref_type=heads.